### DISCUSSION TITLE
I18n support can contribute with spanish and Italian

### DISCUSSION THREAD

**Discussion by johnfelipe** [ghostfolio/ghostfolio#2670] [Ideas]
If you share me strings in Po file or XML or JSON I can translate to Spanish and Italian, is important multilingual feature

**Comment by dtslvr** [comment]
Hi @johnfelipe

You can find the files for localization here: https://github.com/ghostfolio/ghostfolio/tree/main/apps/client/src/locales
`state="new"` means the translation is currently missing.

**Reply by johnfelipe** [reply]
Both lang is already have it
When install locally then will ask you how change lang
Tnks
